draft-ietf-tls-dtls13.md

```diff
@@ -1900,8 +1900,10 @@ MUST be used immediately for all future records. If it is set to
 Endpoints SHOULD use receiver-provided CIDs in the order they were provided.
 Endpoints MUST NOT have more than one NewConnectionId message outstanding.
 
-If the client and server have negotiated the "connection_id" extension,
-either side can request a new CID using the RequestConnectionId message.
+If the client and server have negotiated using the "connection_id" extension,
+either side can request a new CID using the RequestConnectionId message. In
+general, implementations SHOULD use a new CID whenever sending on a new
+path and thus attempt to have enough spare CIDs available to do so.
```

Comment:
This confuses me. Presumably this is trying to say, "implementations should use a new CID for new paths, and should also request new CIDs if they anticipate path changes," or something. 

```suggestion
general, implementations SHOULD use a new CID whenever sending on a new path,
and SHOULD request new CIDs for this purpose if path changes are anticipated.
```